### ISSUE TITLE
Children rule config files

### ISSUE THREAD

**Issue by valaruko** [AtomLinter/linter-eslint#818]
## Issue Type

Question

## Issue Description

I have two different `.eslintrc.json` files one for the root of the project and one with specific rules inside my test directory. eslint seems to be able to figure out which file to use but the IDE linter seems to be ignoring the test config json when linting my test files. Is this configuration supported or am I setting something incorrectly?

## Bug Checklist
- Restart Atom
- Create a new `eslintrc.json` file under a child directory with rule overrides
- Open a file with the child directory and use a rules and use a custom rule from the config file

**Comment by IanVS** [comment]
Thanks for the report! It should be working fine.  Could you provide a few more details, namely:

1. The output of `Linter Eslint: Debug` command when your test file is open.
2. Ideally, a small repo which reproduces this behavior

**Comment by valaruko** [comment]
Thanks for answering, as you said it's working correctly now, probably I had an error in the child `.eslintrc.json` file.

**Comment by deckar01** [comment]
I just spent a good chunk of time trying to figure out why my child config wasn't overriding rules in my parent config. The insane part is that the child config was being parsed and some of the other options were being used correctly. If I included a plugin or env in the child config, linter-eslint would reflect the changes in a source file. Not the rules though. Changes to the rules were being ignored. I restarted Atom and it just started working. This was a frustrating experience.
